src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageQualityGate.java

```diff
@@ -122,5 +143,13 @@ public ListBoxModel doFillCriticalityItems(@AncestorInPath final AbstractProject
             }
             return new ListBoxModel();
         }
+
+        // Used in jelly file.
+        @SuppressWarnings("unused")
+        public String defaultReferenceBuild() {
```

Comment:
I think it is not necessary to use "Localization: Chinese (Simplified)" plugin, since users may not install it.